spec/widgets/annotationsTab.test.js

```diff
@@ -177,16 +177,16 @@ describe('AnnotationsTab', function() {
       spyOn(this.eventEmitter, 'publish');
       subject.bindEvents();
     });
-    it('should deselect a selected item when re-clicked', function() {
-      subject.localState({ selectedList: 'glub1' });
-      subject.element.find('.annotationListItem').first().click();
-      expect(this.eventEmitter.publish).toHaveBeenCalledWith('listDeselected.' + this.windowId, 'glub1');
-    });
-    it('should select an unselected item when clicked', function() {
-      subject.localState({ selectedList: 'glub2' });
-      subject.element.find('.annotationListItem').first().click();
-      expect(this.eventEmitter.publish).toHaveBeenCalledWith('listSelected.' + this.windowId, 'glub1');
-    });
+    // it('should deselect a selected item when re-clicked', function() {
```

Comment:
Should these tests be rewritten to work with new functionality or be config based?